dev/VSIX/overview.md

```diff
@@ -0,0 +1,3 @@
+The Microsoft Windows App SDK is a new set of developer components and tools that represents the next evolution in the Windows app development platform. It provides a unified set of APIs and tools that can be used in a consistent way by any app on a broad set of target Windows 10 OS versions.
+
+This is an experimental version of the Microsoft Windows App SDK. It contains features under development that may appear in future releases.
```

Comment:
Is this the same md that will be included in the release version? If so, please update this language.

Reply:
I've created a stable and preview overview. I've basically copied and pasted over the stable overview from the 0.8 so please take a look.